.github/workflows/test-kfp.yml

```diff
@@ -9,7 +9,6 @@ on:
         tags:
             - "*"
         paths:
-            - ".make.*"
```

Comment:
kfp/Makefile includes .make.defaults, so this should be included.

Reply:
Done. Thanks